packages/type-storage/src/utils/createFunction.ts

```diff
@@ -17,6 +17,18 @@ export interface CreateItemOptions {
   method?: string;
 }
 
+/**
+ * Prepend a Uint8Array with its compact length.
+ *
+ * @param u8a - The Uint8Array to be prefixed
+ */
+function addLengthPrefix (u8a: Uint8Array): Uint8Array {
```

Comment:
Not sure where to put this, we use a couple of times

Reply:
Compact?